<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUploadingWidget</class>
 <widget class="QWidget" name="CUploadingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Выгрузка базы данных</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Выбор концерта</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Поиск</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLineEdit" name="leSearch"/>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="tbnSearch">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../main_res.qrc">
                <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTreeWidget" name="twActions">
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Название мероприятия</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Категория мероприятия</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Выбор даты</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLineEdit" name="leDate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="tbnSelectDate">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="tbClearDate">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../actionsWidget/actionWidget_res.qrc">
             <normaloff>:/images/unavaible.png</normaloff>:/images/unavaible.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnUploading">
       <property name="text">
        <string>Выгрузить</string>
       </property>
       <property name="icon">
        <iconset resource="../main_res.qrc">
         <normaloff>:/images/categoriesImages/imUploading.png</normaloff>:/images/categoriesImages/imUploading.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main_res.qrc"/>
  <include location="../actionsWidget/actionWidget_res.qrc"/>
 </resources>
 <connections/>
</ui>
